cypress/automated-tests/youth-pass/eligibilty-checker-blockers.spec.js

```diff
@@ -0,0 +1,98 @@
+const faker = require('faker');
+
+describe('Eligibility Checker Blockers', () => {
+    const youthPassUrl = Cypress.env('youth_pass_url');
+    const eligibleZipCode = '02114';
+    
+    it('enters an ineligible zip code', () => {
+        const ineligibleZipCode = '01545';
+
+        cy.visit(youthPassUrl);
+        cy.get('#form-section-0 > .form-section-buttons > .form-section-next').click();
+        cy.get('#form-element-wrapper_182').should('not.be.visible');
+
+        cy.get('#element105').type(ineligibleZipCode).blur();
+        cy.get('#form-element-wrapper_182').should('be.visible');
+        cy.get('#form-section-1 > .form-section-buttons > .form-section-next').should('not.be.visible');
+    });
+    
+    it('enters an age below 12', () => {
+        const today = new Date();
+        const currentDate = `${today.getMonth() + 1}/${today.getDate()}/${today.getFullYear()}`;
+        const randomBirthdateTooYoung = faker.date.between('2010-11-01', currentDate);
+        const ineligibleBirthdateTooYoung = 
+            `${randomBirthdateTooYoung.getMonth() + 1}
+            /${randomBirthdateTooYoung.getDate()}
+            /${randomBirthdateTooYoung.getFullYear()}`;
+
+        cy.reload();
+        cy.get('#form-section-0 > .form-section-buttons > .form-section-next').click();
+        cy.get('#element105').type(eligibleZipCode).blur();
+        cy.get('#form-element-wrapper_171').should('not.be.visible');
+
+        cy.get('#element15').type(ineligibleBirthdateTooYoung).blur();
+        cy.get('#form-element-wrapper_171').should('be.visible');
+        cy.get('#form-section-1 > .form-section-buttons > .form-section-next').should('not.be.visible');
+});
+
+    it('enters an age above 25', () => {
+        const randomBirthdateTooOld = faker.date.between('1970-01-01', '1996-10-31');
+        const ineligibleBirthdateTooOld = 
+            `${randomBirthdateTooOld.getMonth() + 1}/
+            ${randomBirthdateTooOld.getDate()}/
+            ${randomBirthdateTooOld.getFullYear()}`;
+        
+        cy.reload();
```

Comment:
I could have gone two ways here: a `reload` command or a `visit` command. I went `reload` because it doesn't materially impact the nature of the test (a page reload always returns the application to a blank slate) and it was 3-5 seconds faster than the `visit` command over the length of the script. Happy to go either way if there's a preference

Reply:
Seems fine to me. If you thought it might be helpful in remembering this later you could always add a comment like "Refreshing this page resets the application to a blank slate" or whatever it might be, but up to you, if you think that would be obvious it's fine to avoid the clutter as well.